src/panels/FlightControlPanel/FlightControlPanel.tsx

```diff
@@ -94,13 +94,25 @@ export function FlightControlPanel() {
       <Title order={3}>Settings</Title>
       <Text>Friction control</Text>
       <Group gap={2} preventGrowOverflow={false} mb={'xs'}>
-        <Chip onClick={toggleRotation} disabled={!isControllerEnabled}>
+        <Chip
```

Comment:
These rotation, zoom, rool chips are now in two places that look almost the same. Maybe they should be their own component?